docs/release-notes/8.4.0-1.md

```diff
@@ -1,2 +1,41 @@
 #  Percona XtraBackup 8.4.0-1 (2024-0-)
 
+Get started with [Quickstart Guide for Percona XtraBackup].
+
+## Release highlights
+
+Percona XtraBackup 8.4.0-1 is based on MySQL 8.4 Long Term Supported (LTS) Release. This release allows taking backups of Percona Server for MySQL and MySQL 8.4.0, 8.4.1, 8.4.2 and future 8.4 LTS releases.
+
+Use the [Percona XtraBackup 8.0 series] to take backups of [Percona Server for MySQL 8.0.x] and MySQL 8.0.x series.
+
+Use the Percona XtraBackup Innovation series, the latest version is [8.3.0-1], to take backups of Percona Server for MySQL Innovation series and MySQL Innovation series.
+
+## Improvements
+
+### Implementations inherited from the Innovation releases
+
+* [PXB-3155](https://perconadev.atlassian.net/browse/PXB-3155): The Percona Server for MySQL 8.0 series implemented the [keyring_vault] plugin. In the Innovation series and 8.4 version, the server shifts to a component infrastructure, the keyring’s functionality remains the same. Percona XtraBackup 8.4.0-1 supports the `keyring_vault` component. Users can back up their encrypted data after they convert from the `keyring_vault` plugin to the `keyring_vault` component.
+
```

Comment:
Alina, Patrick, we have one bug fix that was merged to trunk. I missed this.

Can you please add documentation about PXB-3302

Reply:
Added